sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala

```diff
@@ -38,13 +38,42 @@ object AkkaGrpcPlugin extends AutoPlugin {
       akkaGrpcCodeGenerators := GeneratorAndSettings(ScalaServerCodeGenerator, akkaGrpcCodeGeneratorSettings.value) :: Nil,
       akkaGrpcModelGenerators := Seq[Target]((JvmGenerator("scala", ScalaPbCodeGenerator), akkaGrpcCodeGeneratorSettings.value) -> sourceManaged.value),
 
+      PB.protoSources in Compile += target.value / "protobuf",
+
+      sourceDirectory in PB.recompile := sourceDirectory.value / "protobuf",
+      resourceDirectory in PB.recompile := sourceDirectory.value / "protobuf",
```

Comment:
`PB.protoSources in Compile += target.value / "protobuf"` added here so users don't have to add it on their build.sbt.

`sourceDirectory.value / "protobuf"` hardcoded (or we can switch to `proto`) to prefer convention over config.